UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -78,9 +79,19 @@ public IPlayerPossessable PlayerPossessable
 
 	public PlayerScript ghost { private set; get; }
 	public PlayerScript Body => GetDeepestBody()?.GetComponent<PlayerScript>();
-	private SpawnedAntag antag;
+	private SpawnedAntag antagContainer = null;
 
-	public bool IsAntag => CustomNetworkManager.IsServer ? antag != null : NetworkedisAntag;
+	private SpawnedAntag Antag
+	{
+		get
+		{
+			if (antagContainer == null)
+				antagContainer = GetComponent<SpawnedAntag>();
```

Comment:
if you need to ensure this is available, I would rather cache the component on ``Awake`` than this.